### PR TITLE
[DOC] roadmap

### PR DESCRIPTION
**Pull request by BlazZupan**
Initial commit of the roadmap. Thanks for all contributors in drafting this document.

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Orange Roadmap
+
+In the next few years, we need to expand Orange in the following directions:
```
Comment (ajdapretnar) [inline ROADMAP.md:3]: need to --> plan to, wish to

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+* __Scalability__. Addressing very large data sets in Orange is a challenge. Orange is an interactive data analytic tool with a flexible, user-defined workflow, making optimization for large data more complex than in the case of tools, which are either non-interactive or have a fixed data processing flow. We plan to address the following issues:
```
Comment (ajdapretnar) [inline ROADMAP.md:5]: Addressing --> Processing, handling, analyzing

File: ROADMAP.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+** __Gamification__. We are already exploring ways to implement self-paced tutorials within Orange and design ways to incorporate guidance for workflow construction, problem-driven exercises, and progress badges.
+** __Video content__. We will update and refresh existing [YouTube videos](http://youtube.com/orangedatamining) and record new ones.
+** __Lecture notes__. So far, and for almost a hundred hands-on workshops carried out around the world, we wrote lecture notes. They were typeset in Pages, and while this worked nicely in the beginning, the management of about 300 pages of material became cumbersome. We are converting the lecture notes to LaTeX, enabling fast assembly of notes for a custom-designed hands-on workshop. A current solution needs improvements in design, and text needs updates and additional proofreading.
```
Comment (ajdapretnar) [inline ROADMAP.md:14]: A current solution --> The current solution

File: ROADMAP.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+* __Improved functionality__ in several application fields, with particular priority being
+** __Spectroscopy__, where we are making continuous progress with an add-on, thanks to support from [Synchrotron SOLEIL](https://www.synchrotron-soleil.fr/en).
+** __Text mining__, where our goal is to design components for document characterization, summarization, and explanation of point-based visualizations. This work is in progress, thanks to the support by the [Slovene Ministry of Public Administration](https://www.gov.si/en/state-authorities/ministries/ministry-of-public-administration/).
+** __Network analysis__, where we would like to sped-up and beautify existing visualizations and extend the toolbox with new network analysis procedures.
+** __Time series analysis__. We will replace HiCharts with native visualizations, enable simultaneous analysis of a group of signals, improve modeling procedures, implement model evaluation techniques, and incorporate deep learning and auto-encoding for time series characterization, feature engineering, and prediction.
```
Comment (ajdapretnar) [inline ROADMAP.md:19]: HiCharts --> HighCharts

**Comment by irgolic** [comment]
We should try to make this a bit more maintainable, it reads a bit like an essay at the moment. Drop temporal qualifiers like `In the next few years`. Include instructions for adding new items, so that anyone can contribute (see example https://pandas.pydata.org/docs/development/roadmap.html#roadmap-evolution).

Also, should this be merged with https://github.com/biolab/orange3/issues/4090? If we don't plan to implement something on the big ideas issue, maybe we should drop it.

**Pull request merged**